tests/fixtures/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I don't think this file belongs here. Why did we add it?

Reply:
sorry, I didn't even notice. I've removed it.